editor/plugins/script_text_editor.cpp

```diff
@@ -161,6 +161,15 @@ void ScriptTextEditor::enable_editor() {
 	_enable_code_editor();
```

Comment:
Would it be worth passing it through here too so we only setting it on specific shortcuts? As in should `convert_case` also be tied to the wider script editor? 

Otherwise looks good.

Reply:
The way it currently is is how it works in 3.X. All shortcuts available in the edit menus are usable when find in files is open. Also, I don't think you can set individual items on a popup menu to have a different shortcut context. The shortcuts get routed through the button, so the ctx of the button is applied to the popup menu items.